src/utils.js

```diff
@@ -7,7 +7,7 @@ export const isNode = selector => selector && selector.nodeType === 1
 
 export const isSvg = image => {
   const source = image.currentSrc || image.src
-  return source.substr(-4).toLowerCase() === '.svg'
+  return source.toLowerCase().endsWith(".svg") || source.toLowerCase().includes("/svg/")
```

Comment:
Why should we check for `"/svg/"`? This is not standard.

Reply:
Sometimes the type definition is a url parameter and not the file extension
Ex
SVG file: http://www.plantuml.com/plantuml/svg/SoWkIImgAStDuNBCoKnELT2rKt3AJx9IS2mjoKZDAybCJYp9pCzJ24ejB4qjBk5oICrB0Ie60000

PNG file: http://www.plantuml.com/plantuml/png/SoWkIImgAStDuNBCoKnELT2rKt3AJx9IS2mjoKZDAybCJYp9pCzJ24ejB4qjBk5oICrB0Ie60000